HelpSource/Classes/A2K.schelp

```diff
@@ -8,6 +8,9 @@ Description::
 
 Audio to control rate converter. Only needed in specific cases.
 
+That's the most basic way to make a distinction between Audio and control rate signals are one of them are better to  modulate parameters, wether audio rate signals are ideal to generate audio.
+
+
 
```

Comment:
This isn't the real reason for A2K. Some UGen require a kr control rate for an input. If you need to shove an ar audio rate signal into them then you can use this.

But most UGens (if I remember correctly) can automatically convert audio to control. I think they use the A2K code under the hood, but I would have to go check that to be sure.

So what the helpfile should do is answer what are those "specific cases".

Another use is if you have one audio rate thing (like a microphone input SoundIn.ar which has no .kr version) and wanted to convert to kr and use it in several places.